migration/sql-files/061-replace-index-space-name.sql

```diff
@@ -1,4 +1,14 @@
 -- drop existing unique index
 DROP INDEX spaces_name_idx;
+-- rename duplicate spaces in existence and keep only one as it was
+UPDATE spaces SET name = CONCAT(lower(name), '-renamed')
```

Comment:
I thought another option in case of devil multiple duplicates, to be complete unique. WDYT?